packages/server/postgres/migrations/1625120349057_changeDeletedUserEmail.ts

```diff
@@ -0,0 +1,19 @@
+import {MigrationBuilder, ColumnDefinitions} from 'node-pg-migrate'
+
+export const shorthands: ColumnDefinitions | undefined = undefined
+
+export async function up(pgm: MigrationBuilder): Promise<void> {
+  pgm.sql(`
+     UPDATE "User"
+       SET "email" = CONCAT_WS(':', 'DELETED', "id", TRUNC((extract(epoch from now())*1000)))
+       WHERE "email" = 'DELETED';
+   `)
+}
+
+export async function down(pgm: MigrationBuilder): Promise<void> {
+  pgm.sql(`
+     UPDATE "User"
+       SET "email" = 'DELETED'
+       WHERE "email" LIKE 'DELETED:%';
+   `)
+}
```

Comment:
can you put these as postdeploy migrations instead of predeploy migrations? If we do predeploy, we (1) backfill existing records (2) deploy code with updated write. Whereas with postdeploy, we (1) deploy code with updated write (2) backfill existing records. i think the latter is safer bc there may be a delay btwn predeploy migrations and deployed code running. with dokku, there's a brief period where both old and new container is running before traffic is switched to new.

Reply:
But this has to run before `addUserEmailUniqueConstraint`, right?

Reply:
yes! i meant to put both migration in the postdeploy folder